--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -45,18 +45,18 @@ static CBlock CreateGenesisBlock(uint32_t nTime, uint32_t nNonce, uint32_t nBit
 {
-    const char* pszTimestamp = "The Times 03/Jan/2009 Chancellor on brink of second bailout for banks";
+    const char* pszTimestamp = "The Times 30/Sep/2026 Qitcoin: The Decentralized Trillion Economy";
     const CScript genesisOutputScript = CScript() << ParseHex("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f") << OP_CHECKSIG;
-    return CreateGenesisBlock(pszTimestamp, genesisOutputScript, nTime, nNonce, nBits, nVersion, genesisReward);
+    return CreateGenesisBlock(pszTimestamp, genesisOutputScript, nTime, nNonce, nBits, nVersion, 250000 * COIN);
 }
 
 // In CMainParams constructor:
-genesis = CreateGenesisBlock(1231006505, 2083236893, 0x1d00ffff, 1, 50 * COIN);
-consensus.hashGenesisBlock = genesis.GetHash();
-assert(consensus.hashGenesisBlock == uint256S("0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"));
-assert(genesis.hashMerkleRoot == uint256S("0x4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b"));
+genesis = CreateGenesisBlock(1790772000, 388903, 0x1e0ffff0, 1, 250000 * COIN);
+consensus.hashGenesisBlock = genesis.GetHash();
+assert(consensus.hashGenesisBlock == uint256S("0x0000064d6898cab247cd7a66ae5d621abe9decd284d84b8a1b857b0606dc532a"));
+assert(genesis.hashMerkleRoot == uint256S("0x9a5bcf89553e6cc933455c84a964f720f0aa47628f5bfee08e0db920a37db79a"));
 
 // In CRegTestParams constructor:
-genesis = CreateGenesisBlock(1296688602, 2, 0x207fffff, 1, 50 * COIN);
-consensus.hashGenesisBlock = genesis.GetHash();
-assert(consensus.hashGenesisBlock == uint256S("0x0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206"));
+genesis = CreateGenesisBlock(1790772000, 1, 0x207fffff, 1, 250000 * COIN);
+consensus.hashGenesisBlock = genesis.GetHash();
+assert(consensus.hashGenesisBlock == uint256S("0x3e1630837eda1b40c16c84d5adf7f86b89707a5c63326968f5436c30bc6c2407"));
+assert(genesis.hashMerkleRoot == uint256S("0x9a5bcf89553e6cc933455c84a964f720f0aa47628f5bfee08e0db920a37db79a"));
